### PR TITLE
Add new IcoswISC30E3r6 ocean and sea-ice mesh

### PR DESCRIPTION
**Pull request by xylar**
Long name: IcoswISC30L64E3SMv3r6

This nearly uniform 30 km mesh is the dual mesh of a subdivided icosahedron (Icos).  The mesh is with Ice-Shelf Cavities (wISC) around Antarctica and has 64 vertical layers. It includes smoothing of bathymetry except in ice-shelf cavities over ~100 km.  The ice draft and land-ice thickness are not smoothed (unlike in IcoswISC30E3r4, #6078).

This mesh is a candidate for the E3SM v3 (E3) low res mesh.  It is revision 6 (r6) of the mesh.

The mesh was created using [compass](https://github.com/MPAS-Dev/compass), and is being reviewed there in this PR:
https://github.com/MPAS-Dev/compass/pull/741
As soon as the PR is merged, the code used to produce it will be at this tag: 
https://github.com/MPAS-Dev/compass/releases/tag/mesh_IcoswISC30E3r6

The mesh and simulation results (to follow) are been reviewed and approved here:
https://acme-climate.atlassian.net/wiki/spaces/OO/pages/4021223426/Review+IcoswISC30E3r6

A B-case will begin shortly and analysis will be posted here and on the review page as soon as it is available.

### CODE REVIEW

**Comment by proteanplanet** [comment]
An brief summary of the ability of this mesh to address polar questions is provided here: https://github.com/MPAS-Dev/compass/pull/741#pullrequestreview-1760948789

A full analysis of the mesh for critical ocean passages, coastal bathymetry, and ice shelves is available at the bottom of this page: https://acme-climate.atlassian.net/wiki/spaces/OO/pages/4021223426/Review+IcoswISC30E3r6

**Comment by jonbob** [comment]
I added the necessary mapping and domain files, and then ran a G-case for a month to spin-up some initial conditions. After that I started a B-case with those spun-up ICs. As a reminder, this failed in the r4 version and in the end we needed to run the G-case for a year in order for the B-case to run successfully.

**Comment by xylar** [comment]
@darincomeau, @stephenprice and @alicebarthel, I removed you as reviewers to save some time, not because I don't want your input. @golaz would like to have this PR on master before a control run starts with this mesh so I'm trying to reduce the number of reviewers to accelerate that process.

**Comment by xylar** [comment]
@vanroekel and @proteanplanet, same for this mesh, please review and hopefully approve when you can.  Let me know if I can do something to help with that process.

@maltrud, I'd also really like to have your review.  My hope is that this is as close as the Compass workflow can get to what you did in your simulations.  There isn't really time for another iteration as you know but it still would be worthwhile for you to point out differences you notice between your mesh and this one.

**Review by proteanplanet — APPROVED** [review]
Inasmuch as this mesh is technically able to be run in E3SM and correctly generated, I approve this PR.  

Scientifically I cannot recommend use of this mesh in E3SM based on results already obtained with its predecessor with the same level of smoothing, except for under ice shelves. That is based on a comparison provided here:

A: Bathymetry important for ocean simulation: 1) Continental shelves removed along the Western Americas, around Antarctic, Africa, and East Australia; 2) Near complete removal of the Aleutian island chain and associated shallow bathymetry; 3) Removal of deep trenches and critical banks. 

MPAS_OceanIce_IcoswISC30E3r2_IcoswISC30E3r6_sector_bathymetry_1

MPAS_OceanIce_IcoswISC30E3r2_IcoswISC30E3r6_sector_bathymetry_6

MPAS_OceanIce_IcoswISC30E3r2_IcoswISC30E3r6_sector_bathymetry_7


B: AMOC is unimproved with the predecessor to this mesh that differs only by smoothing under ice shelves:

MOC_Comparison


C: There is an undesirable dynamical change associated with this mesh to the barotropic mode, especially in the Southern Ocean:

Icos_Barotropic_Wave_Speed_Smoothed_Cropped

**Review by vanroekel — APPROVED** [review]
I'm making a similar review to @proteanplanet -- this implementation is correct and produces what we expect.  But will have to see very clear improvements to recommend this as the mesh for v3 coupled model.

**Comment by xylar** [comment]
I realize there's a good chance this won't go to next today but I rebased just in case.  I'm about to head out for the day.

**Comment by vanroekel** [comment]
All, based on significant analysis and simulation I want to recommend that we close this PR without merging.  the benefits of smoothing in coupled cases is minor and this version of smoothing is not something we want used long term and thus don't think we want in the model.  Therefore I would advocate we not merge this PR.

**Comment by xylar** [comment]
Okay, thanks @vanroekel.  I'm closing based on your recommendation.

**Pull request closed without merging**
